spec/checkPullRequestJobSpec.js

```diff
@@ -0,0 +1,410 @@
+    additions: 1,
+    deletions: 0,
+    changes: 1,
+    blob_url:
+      'https://github.com/oppia/oppia/blob/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_one_off.py',
+    raw_url:
+      'https://github.com/oppia/oppia/raw/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_one_off.py',
+    contents_url:
+      'https://api.github.com/repos/oppia/oppia/contents/core/domain/exp_jobs_one_off.py?ref=67fb4a973b318882af3b5a894130e110d7e9833c',
+    patch: '@@ -0,0 +1 @@\n+# Testing job pushes',
+  };
+
+  const secondNewJobFileObj = {
+    sha: 'd144f32b9812373d5f1bc9f94d9af795f09023ff',
+    filename: 'core/domain/exp_jobs_oppiabot_off.py',
+    status: 'added',
+    additions: 1,
+    deletions: 0,
+    changes: 1,
+    blob_url:
+      'https://github.com/oppia/oppia/blob/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_oppiabot_off.py',
+    raw_url:
+      'https://github.com/oppia/oppia/raw/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_oppiabot_off.py',
+    contents_url:
+      'https://api.github.com/repos/oppia/oppia/contents/core/domain/exp_jobs_oppiabot_off.py?ref=67fb4a973b318882af3b5a894130e110d7e9833c',
+    patch: '@@ -0,0 +1 @@\n+# Testing job pushes',
+  };
+
+  const registryFileObj = {
+    sha: 'd144f32b9812373d5f1bc9f94d9af795f09023ff',
+    filename: 'core/jobs_registry.py',
+    status: 'modified',
+    additions: 1,
+    deletions: 0,
+    changes: 1,
+    blob_url:
+      'https://github.com/oppia/oppia/blob/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_oppiabot_off.py',
+    raw_url:
+      'https://github.com/oppia/oppia/raw/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_oppiabot_off.py',
+    contents_url:
+      'https://api.github.com/repos/oppia/oppia/contents/core/domain/exp_jobs_oppiabot_off.py?ref=67fb4a973b318882af3b5a894130e110d7e9833c',
+    patch: '@@ -0,0 +1 @@\n+# exp_jobs_oppiabot_off exp_jobs_one_off',
+  }
+
+  beforeEach(() => {
+    spyOn(scheduler, 'createScheduler').and.callFake(() => {});
+
+    github = {
+      issues: {
+        createComment: jasmine.createSpy('createComment').and.returnValue({}),
+        addLabels: jasmine.createSpy('addLabels').and.returnValue({}),
+        addAssignees: jasmine.createSpy('addAssignees').and.returnValue({})
+      },
+    };
+
+    robot = createProbot({
+      id: 1,
+      cert: 'test',
+      githubToken: 'test',
+    });
+
+    app = robot.load(oppiaBot);
+    spyOn(app, 'auth').and.resolveTo(github);
+    spyOn(checkPullRequestJobModule, 'checkForNewJob').and.callThrough();
+    spyOn(apiForSheetsModule, 'checkClaStatus').and.callFake(() => {});
+    spyOn(
+      checkPullRequestLabelsModule,
+      'checkChangelogLabel'
+    ).and.callFake(() => {});
+    spyOn(checkPullRequestBranchModule, 'checkBranch').and.callFake(() => {});
+    spyOn(checkWIPModule, 'checkWIP').and.callFake(() => {});
+  });
+
+  describe('When pull request creates a new job file', () => {
```

Comment:
`When pull request creates a new job file` -> `When a new job file is created in a pull request`